### ISSUE TITLE
Nuxt/Prisma conflict with Nuxt/UI

### ISSUE THREAD

**Issue by DT-Zilo** [prisma/nuxt-prisma#64]
### Environment

- OS : Windows 11
- Node : 20.14.0 
- Nuxt: lastest
- Nuxt prisma : lastest

### Version

2.19.1

### Reproduction

init lastest nuxt 3
add nuxt prisma module
add nuxt ui module

` ERROR  RollupError: node_modules/@prisma/nuxt/dist/runtime/server/utils/prisma.d.ts (2:8): Expected ';', '}' or <eof> (Note that you need plugins to import files that are not JavaScript)                       nitro 21:27:12  

1: import { PrismaClient } from "@prisma/client";
2: declare const prismaClientSingleton: () => PrismaClient<import(".prisma/client").Prisma.PrismaClientOptions, never, i...
           ^
3: declare const prisma: PrismaClient<import(".prisma/client").Prisma.PrismaClientOptions, never, import("@prisma/client...
4: export type CustomPrismaClient = ReturnType<typeof prismaClientSingleton>;`

### Description

when remove `'@nuxt/ui'` from modules: [''] in nuxt.config.ts it's work can use npm run dev
but if add `'@nuxt/ui'` it''s error

` ERROR  RollupError: node_modules/@prisma/nuxt/dist/runtime/server/utils/prisma.d.ts (2:8): Expected ';', '}' or <eof> (Note that you need plugins to import files that are not JavaScript)                       nitro 21:27:12  

1: import { PrismaClient } from "@prisma/client";
2: declare const prismaClientSingleton: () => PrismaClient<import(".prisma/client").Prisma.PrismaClientOptions, never, i...
           ^
3: declare const prisma: PrismaClient<import(".prisma/client").Prisma.PrismaClientOptions, never, import("@prisma/client...
4: export type CustomPrismaClient = ReturnType<typeof prismaClientSingleton>;`

**Comment by gsxdsm** [comment]
Same issue here

**Comment by gsxdsm** [comment]
Related to #12

Deleting node_modules/@prisma/nuxt/dist/runtime/server/utils/prisma.d.ts temporarily solves it, but it's not a long term solution. @ankur-arch - can this be prioritized? It is a huge annoyance.

**Comment by gsxdsm** [comment]
Of course, deleting prisma.d.ts means you lose types and IDE autocomplete... :-(

**Comment by LarsDD-gh** [comment]
Got the same issue.

Any idea, how to work-arround it? Maybe by configuring rollup to skip the problematic file?

**Comment by ankur-arch** [comment]
@LarsDD-gh 

> Any idea, how to work-arround it? Maybe by configuring rollup to skip the problematic file?

Could you folks see if this version works for you? `0.0.31-fix-rollup-error`. As, this seems like a regression to me.

Thanks, @gsxdsm! It’s been two weeks of hectic planning, so I’ve been incredibly busy. I really appreciate all your insights and completely understand the frustration.

I’ll make room and prioritize this next week!

**Comment by LarsDD-gh** [comment]
@ankur-arch sorry for beeing "uninformed" ;) but where do i find this fix? I did not see it in the available tags for this repo :/

**Comment by DesertCookie** [comment]
> Could you folks see if this version works for you? `0.0.31-fix-rollup-error`.

Works for me after downgrading from `"@prisma/nuxt": "0.1.2"`. Thanks!

**Comment by gsxdsm** [comment]
Thanks so much @ankur-arch ! Testing today and will report back but looks promising.

**Comment by ankur-arch** [comment]
Hi @gsxdsm, let me know if the Rollup error is being addressed. I’ll push an update once it’s resolved—it definitely seems like a regression.

**Comment by gsxdsm** [comment]
Yes seems to be resolved thank you!

**Comment by andreakru** [comment]
does this still work only with `0.0.31-fix-rollup-error` ?

**Comment by ankur-arch** [comment]
Hi @andreakru and @ThomasBerne , I'm trying to identify the regression and have a pre-release version `@prisma/nuxt@0.0.0-experimental-72b9dd7` that I haven't been able to test completely. 

Can you see if the provided version works, then I'll merge it in.

**Comment by andreakru** [comment]
> Hi @andreakru and @ThomasBerne , I'm trying to identify the regression and have a pre-release version `@prisma/nuxt@0.0.0-experimental-72b9dd7` from this [PR](https://github.com/prisma/nuxt-prisma/pull/68) that I haven't been able to test completely.
> 
> Can you see if the provided version works, then I'll merge it in.

It seems to works fine for me.

**Comment by ankur-arch** [comment]
Thanks for your patience everyone, I've merged it in 🥳 , and this issue should be fixed in the latest version v`0.1.3`!
